libs/execution_context/ebpf_maps.c

```diff
@@ -2125,6 +2129,16 @@ ebpf_map_delete_entry(_In_ ebpf_map_t* map, size_t key_size, _In_reads_(key_size
             map->ebpf_map_definition.key_size);
         return EBPF_INVALID_ARGUMENT;
     }
+
+    if (ebpf_map_metadata_tables[map->ebpf_map_definition.type].delete_entry == NULL) {
+        EBPF_LOG_MESSAGE_UINT64(
+            EBPF_TRACELOG_LEVEL_ERROR,
+            EBPF_TRACELOG_KEYWORD_MAP,
+            "ebpf_map_delete_entry not supported on map",
+            map->ebpf_map_definition.type);
+        return EBPF_INVALID_ARGUMENT;
```

Comment:
minor: should this be `EBPF_OPERATION_NOT_SUPPORTED` instead?

Reply:
@saxena-anurag fixed it, can you please re-approve?